### ISSUE TITLE
Reduce boilerplate for stat/inventory evaluation in Redfish

### ISSUE THREAD

**Issue by matejart** [xlab-si/wrapanapi#1]
```python
def rack_stats(self, ...):
  return evaluate_lambdas(hash_with_lambdas, keys, lambda_args)

def rack_inventory(self, ...):
  return evaluate_lambdas(hash_with_lambdas, keys, lambda_args)

def chassis_stats(self, ...):
  return evaluate_lambdas(hash_with_lambdas, keys, lambda_args)

def chassis_inventory(self, ...):
  return evaluate_lambdas(hash_with_lambdas, keys, lambda_args)
```
(of course, you need to provide correct values for `hash_with_lambdas`, `keys` and `lambda_args`)

Because we're doing the same thing all the time, evaluating correct lambdas... Maybe we could put `evaluate_lambdas` in a Mixin even to make it reusable.

_Originally posted by @miha-plesko in https://github.com/ManageIQ/wrapanapi/pull/335_

**Comment by matejart** [comment]
Moved the issue to the [private repository](https://github.com/xlab-si/wrapanapi/issues/1).

**Comment by miha-plesko** [comment]
Yeah it feels like there is such mixin implemented somewhere in the project already unless there's a good reason why it's not possible. Therefore I'd include @mshriver in the debate to get an expert opinion on this.
